frontend/src/router.ts

```diff
@@ -155,7 +179,21 @@ const router = createRouter({
 });
 
 router.beforeEach((to, from) => {
-  if (!to.meta?.isPublic && !['setup', 'contact', undefined].includes(to.name)) {
+  const toMeta: ApmtRouteMeta = to?.meta ?? {};
+  const fromMeta: ApmtRouteMeta = from?.meta ?? {};
+  const usePosthog = inject(usePosthogKey);
```

Comment:
I get the following console warning:

![image](https://github.com/user-attachments/assets/e264894b-d571-4d94-ba8f-245caeef6a8d)

I'm not 100% sure if this is the cause, but this is the only place an inject was added...

Reply:
Thank you! Have a nice weekend!